files/KoreBuild/scripts/KoreBuild.psm1

```diff
@@ -431,7 +423,9 @@ function __get_dotnet_sdk_version {
         Write-Warning "dotnet SDK version overridden by KOREBUILD_DOTNET_VERSION"
         return $env:KOREBUILD_DOTNET_VERSION
     }
-    return Get-Content (Join-Paths $PSScriptRoot ('..', 'config', 'sdk.version'))
```

Comment:
Since we don't generate global.json anymore, let's also remove usages of KOREBUILD_DOTNET_VERSION, both in .ps1 and .sh scripts.